src/services/ledger/helpers.ts

```diff
@@ -1,4 +1,5 @@
-import { UnknownLedgerError } from "./domain/errors"
+import { UnknownLedgerError } from "@domain/ledger"
```

Comment:
I was not part of ledger refactor but I guess this was on purpose @UncleSamtoshi @bodymindarts could you please check this?

Reply:
Relevant commit here for @UncleSamtoshi / @bodymindarts
https://github.com/GaloyMoney/galoy/pull/1600/commits/52b248d351e7ad140d985fb720c4a735fdd92425

Reply:
I dont recall the reason for it and think it would be fine to change. Perhaps @bodymindarts has an opinion on it.

Reply:
We want to isolate the ledger internally. It should not depend on @domain. Rather use its own domain types. Exception is dependencies on shared